schema/vrs.yaml

```diff
@@ -184,10 +184,37 @@ definitions:
         oneOf:
           - $ref: "#/definitions/MolecularVariation"
           - $ref: "#/definitions/Feature"
-          - $ref: "#/definitions/Location"
+          - $ref: "#/definitions/SequenceExpression"
```

Comment:
Please leave Location in until we agree to remove it.

Without Location, how would we refer to CopyNumber of a region? Via a DerivedSequenceExpression at a Location?

Reply:
Yes, via `DerivedSequenceExpression`. All changes in this draft PR are open to discussion; not put here as the final solution, only something to be discussed and resolved for #285.

Reply:
Using `Location` feels off to me and I think that `SequenceExpression` classes provide more extensibility when it comes to systemic copies of sequences. I do not feel strongly about this, but I would like to avoid a situation where we allow either Location OR SequenceExpression, and I see the latter as more extensible.

Reply:
Agreed. Will merge.